### PR TITLE
add more tmux details

### PR DESCRIPTION
**Pull request by setharnold**
A fellow cve consumer from netbsd has asked for some more details in the description and another url in the references. Most of the url noise is just vulnogram reordering json fields.

### CODE REVIEW

**Comment by cristina479** [comment]
@setharnold Please resolve the conflicts so that we can publish the changes of CVE-2020-27347.

**Comment by setharnold** [comment]
Hello @cristina479 , I believe I've done so. Thanks

File: 2020/27xxx/CVE-2020-27347.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
         "description_data": [
```
Comment (cve-team) [inline 2020/27xxx/CVE-2020-27347.json:67]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2020/27xxx/CVE-2020-27347.json:67]: ("Expecting ',' delimiter: line 120 column 17 (char 4288)",)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Pull request merged**